addons/event/models/event_event.py

```diff
@@ -82,6 +82,26 @@ def _onchange_has_seats_limitation(self):
             self.default_registration_min = 0
             self.default_registration_max = 0
 
+class EventCategory(models.Model):
+    """Event Category"""
+    _name = "event.tag.category"
+    _description = "Event Category"
+    _order = "sequence"
+
+    name = fields.Char("name", required=True)
+    sequence = fields.Integer('Sequence', default="0")
```

Comment:
I see default=1 and default=10, but assigning a string value of "0" sounds like a bad idea.